tests/MongoDB.Analyzer.Tests.Common/DiagnosticTestCaseAttribute.cs

```diff
@@ -13,31 +13,43 @@
 // limitations under the License.
 
 using System;
-
 namespace MongoDB.Analyzer.Tests.Common
 {
+    [Serializable]
+    public record Location(
```

Comment:
1. Lets extract this to its own Location.cs file.
2. The closing bracket should be in same line as last parameter.
3. Do we need it to be serializable?
4. This is property name, so pascal case (as opposed to argument)

Reply:
1. Done, 2. Done, 3. it has to be serializable since the Diagnostic Rule record is Serializable and it contains a Location attribute. 4. Done